tutorials/sensor/lidar_node.cc

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <iostream>
+#include <string>
+#include <ignition/msgs.hh>
+#include <ignition/transport.hh>
+
+ignition::transport::Node node;
+std::string topic_pub = "/cmd_vel";
+ignition::msgs::Twist data;
+auto pub = node.Advertise<ignition::msgs::Twist>(topic_pub);
```

Comment:
How about defining all these inside the `main`? It's always good to avoid global variables.

Reply:
You are right. My intention was to make it simple.
there are two solutions in my mind right now. 
* these variables inside the callback function. but this will be initialized with every call.
* we can make a class.

Reply:
I think we shouldn't need a class. The node, publisher and topic can be declared in `main`, and data can be declared in the callback. It's fine for the scope of these variables to be limited to those functions because they're only used there.

Reply:
I think the publisher need to be defined inside the callback. because at the end of the callback I publish the data and in this case the node also need to be defined inside the callback.
So I suggest having the node globaly defined instead of defining it twice in the main and in the callback